dansCarRental/carRental/templates/carRental/account.html

```diff
@@ -1,81 +1,90 @@
 {% extends 'base.html' %}
 
-{% block title %}Home{% endblock %}
+{% block title %}Account{% endblock %}
 
 {% load static %}
 {% block content %}
 {% if user.is_authenticated %}
 
-
+<form action="{% url 'account' %}" method="post">
```

Comment:
I would consider splitting this form up into each button, if not a user may submit a form they did not intend to.